graylog2-server/src/main/java/org/graylog2/plugin/Version.java

```diff
@@ -217,6 +243,12 @@ public static Version fromClasspathProperties(String path, String propertyName,
         return defaultVersion;
     }
 
+    private static URL getResource(Class<?> clazz, String path) {
+        final URL url = requireNonNull(clazz, "Class argument is null!").getClassLoader().getResource(path);
```

Comment:
I think the `getClassLoader()` call is not required here, as [`Class#getResource(String)`](https://docs.oracle.com/javase/8/docs/api/java/lang/Class.html#getResource-java.lang.String-) already is using the object's class loader.

There's also a variant of [`Resources#getResource()`](https://google.github.io/guava/releases/snapshot/api/docs/com/google/common/io/Resources.html#getResource(java.lang.Class,%20java.lang.String)) for this.